website/snippets/_cloud-environments-info.md

```diff
@@ -28,11 +28,30 @@ Both development and deployment environments have a section called **General Set
 | Default to Custom Branch | ☑️ | Determines whether to use a branch other than the repository’s default  | See below |
 | Custom Branch | dev | Custom Branch name | See below |
 
-:::note About dbt version
+### About versions in dbt Cloud
 
-- dbt Cloud allows users to select any dbt release. At this time, **environments must use a dbt version greater than or equal to v1.0.0;** [lower versions are no longer supported](/docs/dbt-versions/upgrade-core-in-cloud).
-- If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
-:::
+dbt Cloud allows users to select from multiple dbt releases. At this time, environments must use a dbt version greater than or equal to v1.0.0 [lower versions are no longer supported](/docs/dbt-versions/upgrade-core-in-cloud).
+If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
+
+
+### Git repository caching (Enterprise plans only)
+
```

Comment:
i agree the 'enterprise only' callout should be higher than it currently is. however, I don't think we should add 'enterprise plan only' as part of the header and URL slug. I don't think we should add descriptive words like this in the header because it'll become part of the URL anchor link and can cause 'page not found' URL issues when we eventually add the plan pill component. 

![Screenshot 2024-01-09 at 11 44 39](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/db4b44c9-e889-4319-8c2e-daa2cd66acb7)

also the image 'Example of the Repository caching option' isn't rendering ?

```suggestion
:::info Available for dbt Cloud Enterprise plans
This feature is exclusively available for dbt Cloud Enterprise plans. [Book a demo](https://www.getdbt.com/contact) to learn more.
:::

```